scripts/test_commands.sh

```diff
@@ -0,0 +1,86 @@
+# EXAMPLE:
+# ./scripts/sanity_check.sh -c IntegrationTestCluster -r us-east-2
```

Comment:
Hmm, am I missing something? I don't see a `sanity_check.sh` file anywhere in the `/scripts` directory.

Reply:
oops, I changed the name